static/css/style.css

```
@@ -243,6 +244,14 @@ img {
     margin-top: 0;
 }
 
+.main .page-variant-registration ~ pre, .main .page-variant-registration ~ * pre {
+    padding: .5em .75em;
+    border-radius: .25em;
+    background: var(--c-off-white);
+    color: var(--c-hl-dark);
+    box-shadow: 0 0 1px  var(--c-bg-light);
```

I'd prefer if this was
```suggestion
    border: 1px solid var(--border-color);
```

---

Applied the border to `pre`, but not `code` (see [this comment](https://github.com/freenode/web-7.0/pull/419#discussion_r312715552))